src/com/owncloud/android/ui/fragment/ExtendedListFragment.java

```diff
@@ -228,7 +273,11 @@ public void onActivityCreated(Bundle savedInstanceState) {
             mTops = new ArrayList<Integer>();
             mHeightCell = 0;
         }
-    }    
+
+        SharedPreferences appPreferences = PreferenceManager
+                .getDefaultSharedPreferences(MainApp.getAppContext());
```

Comment:
To minimize dependencies you could use getActivity() as a context. Or is there any specific reason to use application context here ?